ios/KochavaPlugin.mm

```diff
@@ -67,8 +69,7 @@ - (void) trackPurchase:(NSDictionary *)jsonObject {
     if(jsonData) {
       NSString *jsonString = [[NSString alloc] initWithData:jsonData
                                                encoding:NSUTF8StringEncoding];
-      [self.kochavaTracker trackEvent:@"Purchase" withValue:jsonString
-                      andReceipt:[jsonObject valueForKey:@"receipt"]];
+      [self.kochavaTracker sendEventWithNameString:@"Purchase" infoString:jsonString];
```

Comment:
Please use default event name here. Please refer https://support.kochava.com/sdk-integration/ios-sdk-integration/ios-using-the-sdk/?scrollto=marker_9

Reply:
@rampr 

You mean this
**Standard Event with Standard Parameters**
```
KochavaEvent *event = [KochavaEvent eventWithEventTypeEnum:KochavaEventTypeEnumPurchase];
event.nameString = @"Gold Token";
event.priceDoubleNumber = @(0.99);
[KochavaTracker.shared sendEvent:event];
```

Reply:
yes @anil-kumar07